src/auth/users.controller.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Post,
+  ValidationPipe,
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { CreateUserDto } from './input/create.user.dto';
+import { User } from './entities/user.entity';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+
+@Controller('users')
+export class UsersController {
+  constructor(
+    private readonly authService: AuthService,
+    @InjectRepository(User)
+    private readonly userRepository: Repository<User>,
+  ) {}
+
+  @Post()
+  async create(@Body(new ValidationPipe()) createUserDto: CreateUserDto) {
+    const user = new User();
+
+    if (createUserDto.password !== createUserDto.confirmPassword) {
+      throw new BadRequestException(['Passwords are not identical!']);
```

Comment:
Please check if we can skip the array and send only a string.

Reply:
Removed